koku/masu/database/trino_sql/gcp/openshift/reporting_ocpgcp_cost_summary_by_service_p.sql

```diff
@@ -0,0 +1,49 @@
+-- Clear out old entries first
```

Comment:
(super nitpick): we aren't doing the deletes in this SQL file, should probably remove this comment